### PR TITLE
[Cloud Security] [CNVM] Added mappings to vulnerability

### PR DESCRIPTION
**Pull request by kfirpeled**
## What does this PR do?

Added field mapping to vulnerability and data views

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      type: keyword
+    - name: data_source.ID
+      type: keyword
+    - name: data_source.URL
+      type: keyword
+    - name: data_source.Name
+      type: keyword
+    - name: cwe
+      type: keyword
+    - name: score.impact
```
Comment (uri-weisman) [inline packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:20]: IMO, the impact field can be omitted until we get a better understanding if it's needed. (https://github.com/elastic/security-team/issues/6340#issuecomment-1511504755)
Currently, it's not being calculated the right way.

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+- name: vulnerability
+  type: group
```
Comment (uri-weisman) [inline packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:2]: In the ecs file, I would add the cloud-related fields that were added as part of this [task](https://github.com/elastic/cloudbeat/pull/864#issue-1669931393).
Similar to the cloud fields that are part of cspm + the region.

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      type: keyword
+    - name: package.fixed_version
+      type: keyword
+    - name: title
+      type: keyword
+    - name: data_source.ID
+      type: keyword
+    - name: data_source.URL
+      type: keyword
+    - name: data_source.Name
```
Comment (opauloh) [inline packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:16]: Do we still have time to convert those fields to lowercase?

**Review by opauloh — APPROVED** [review]
LGTM - just comments that can be addressed later

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml (hunk 2 above)
Comment (kfirpeled) [reply packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:2]: @uri-weisman these are the cloud related fields currently available both in `findings` and `vulnerabilities` datastreams:
```yaml
- name: cloud.account.id
  external: ecs
- name: cloud.account.name
  external: ecs
- name: cloud.provider
  external: ecs
```

Can you elaborate what you referred to, that you'd like us to add?

Comment (kfirpeled) [reply packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:2]: is it only `cloud.region`?

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml (hunk 3 above)
Comment (kfirpeled) [reply packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:16]: I believe @elastic/cloudbeat should answer this

cc: @oren-zohar , @olegsu , @jeniawhite

File: packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml (hunk 2 above)
Comment (uri-weisman) [reply packages/cloud_security_posture/data_stream/vulnerabilities/fields/vulnerability.yml:2]: @kfirpeled 
Yes, the fields you mentioned + `cloud.region`
```yaml
- name: cloud.account.id
  external: ecs
- name: cloud.account.name
  external: ecs
- name: cloud.provider
  external: ecs
- name: cloud.region
  external: ecs
```

**Pull request merged**